app/main/menu.ts

```diff
@@ -4,7 +4,7 @@ import * as path from 'path';
 import { app, shell, BrowserWindow, Menu, dialog } from 'electron';
 
 import * as fs from 'fs-extra';
-import * as AdmZip from 'adm-zip';
+import AdmZip from 'adm-zip';
```

Comment:
@priyank-p Where is this documented that we can import it this way? I suggest a better way would be to do all imports with commonjs syntax as ```import x = require('x')``` .This won't cause any problems.

Reply:
@vsvipul If you feel strong about using commonjs import syntax feel free to update this :). The `import AdmZip from 'adm-zip';` is a ES6 default import statement meaning if the `amd-zip` module does `export default` or `module.exports.default` we will be able to import it. The module do that as we know, but typescript adds `__importDefault` helper which takes care of this.

Reply:
Ok, will change to commonjs syntax then.